gotham-server/src/lib.rs

```diff
@@ -6,9 +6,9 @@
 // License as published by the Free Software Foundation, either
 // version 3 of the License, or (at your option) any later version.
 //
-
-#![feature(proc_macro_hygiene, decl_macro)]
-
+#![recursion_limit="128"]
```

Comment:
Can you explain this line?

Reply:
It solves this error when using `error-chain`:
```
error: recursion limit reached while expanding the macro `derive(Debug)`
  --> src/storage/aws/error.rs:5:1
   |
5  | / error_chain! {
6  | |     foreign_links {
7  | |         DescribeTable(rusoto_dynamodb::DescribeTableError);
8  | |         ListTables(rusoto_dynamodb::ListTablesError);
...  |
42 | |     }
43 | | }
   | |_^
   |
   = help: consider adding a `#![recursion_limit="128"]` attribute to your crate
   = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)
```